WcaOnRails/spec/factories/results.rb

```diff
@@ -36,15 +43,38 @@
       value5 { 0 }
     end
 
-    trait :blind_dnf_mo3 do
+    trait :blind_mo3 do
+      mo3
       eventId { "333bf" }
       formatId { "3" }
+    end
+
+    trait :blind_dnf_mo3 do
+      blind_mo3
       average { -1 }
-      value1 { best }
-      value2 { best }
       value3 { -1 }
-      value4 { 0 }
-      value5 { 0 }
     end
+  }
+
+  factory :inbox_result do
+    instance_eval(&resultable_instance_members)
```

Comment:
I couldn't find a "better" way to DRY the attributes initializers and traits common to `Result` and `InboxResult` :s